climpred/tests/test_hindcast_prediction.py

```diff
@@ -1,17 +1,17 @@
 import numpy as np
 import pytest
-
 from climpred.bootstrap import bootstrap_hindcast
-
-# testing less separately: use PM_METRICS
-from climpred.constants import HINDCAST_COMPARISONS, PM_METRICS
+from climpred.constants import DETERMINISTIC_HINDCAST_METRICS, HINDCAST_COMPARISONS
 from climpred.prediction import (
     compute_hindcast,
     compute_persistence,
     compute_uninitialized,
 )
 from climpred.tutorial import load_dataset
 
+# uacc is sqrt(ppp), fails when ppp negative
+DETERMINISTIC_HINDCAST_METRICS.remove('uacc')
```

Comment:
I think this modifies the original object inplace meaning that everywhere else will be missing uacc.